### DISCUSSION TITLE
All Kibana Visualizations are erroring after upgrade to 2.3.40

### DISCUSSION THREAD

**Discussion by wtakang** [Security-Onion-Solutions/securityonion#3832] [Unsupported Versions]
After we upgraded to SO 2.3.40, all kibana visualizations are erroring out when you click on them. What we get instead is this message: 
"There is a problem with this saved object
A field associated with this object no longer exists in the index pattern.
If you know what this error means, go ahead and fix it — otherwise click the delete button above."
Then below that there is this other warning: 
"Proceed with caution!
Modifying objects is for advanced users only. Object properties are not validated and invalid objects could cause errors, data loss, or worse. Unless someone with intimate knowledge of the code told you to be in here, you probably shouldn’t be.". 

For every visualization, there is a missing field for id. 
any thoughts?

**Comment by weslambert** [comment]
You might try running `so-kibana-config-load` to see if that helps.
